packages/scandipwa/src/component/FieldSelect/FieldSelect.component.js

```diff
@@ -96,11 +96,18 @@ export class FieldSelect extends PureComponent {
             handleSelectListOptionClick
         } = this.props;
 
+        const isDisabled = option.isAvailable !== undefined ? !option.isAvailable : false;
```

Comment:
Please destruct isAvailable from option and set default value for it, so you could just pass it to mods without checking if it is undefined.